config/default-openshift/kustomization.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# Adds namespace to all resources.
+namespace: vault-secrets-operator-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: vault-secrets-operator-
+
+# Labels to add to all resources and selectors.
+#commonLabels:
+#  someName: someValue
+
+bases:
+- ../crd
```

Comment:
Would it be possible to replace all of these bases with `../default`

Reply:
Yep, good call, updated in d1c2c8a